scripts/build.sh

```diff
@@ -3,8 +3,23 @@
 PROJECT_ROOT="$(dirname "$(dirname "$(readlink -f "$0")")")"
 cd "${PROJECT_ROOT}" || exit 1
 
+# Make sure no one is souring the script before we change GOBIN
+if [[ "${BASH_SOURCE[0]}" != "$0" ]]; then
+	echo "Execute ${BASH_SOURCE[0]} instead of souring it"
+	return
+fi
+
+# Fail script on any error
+set -e
+
+mkdir -p bin
+export GOBIN="${PROJECT_ROOT}/bin"
+
 if [[ -n "${ALL}" ]]; then
-	GOBIN="${PROJECT_ROOT}" go install -tags=prod ./cmd/...
+	go install -tags=prod ./cmd/...
 else
-	GOBIN="${PROJECT_ROOT}" go install -tags=prod ./cmd/swapd ./cmd/swapcli
+	go install -tags=prod ./cmd/swapd ./cmd/swapcli
 fi
+
+# Will change the suffix below to @latest before merging
+go install github.com/athanorlabs/go-relayer/cmd/relayer@95cca1a2b8b2b16a78f9a0d8fa367cf2e9c95a4f
```

Comment:
I'll remove the commit hash after we merge this PR:
https://github.com/AthanorLabs/go-relayer/pull/6

Note: Normally you could put a branch name after the `@` instead of a commit hash which requires more frequent updating, but they don't allow you to have a slash in the branch name and I was using `dimalinux/key-handling`.